yapapi/payload/vm.py

```diff
@@ -222,40 +214,38 @@ async def repo(
         )
 
     """
+
+    repo_url = ApiConfig().repository_url
```

Comment:
again, repository url shouldn't be supplied in ApiConfig as it's clearly documented as a way to attach to a particular instance of yagna - not as the configuration of _this_ library...